### PR TITLE
Post-preapproval stuffs

### PR DESCRIPTION
**Pull request by blinkseb**
See list of commits

### CODE REVIEW

File: histFactory_hh/basePlotter.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def __init__(self, baseObjectName = "hh_llmetjj_allTight_btagM_csv", btagWP_str
     def generatePlots(self, categories = ["All"], stage = "cleaning_cut", requested_plots = [], weights = ['trigeff', 'jjbtag', 'llidiso', 'pu'], extraCut = "", systematic = "nominal", extraString = "", fit2DtemplatesBinning = None):
 
         # MVA evaluation : ugly but necessary part
-        baseStringForMVA_part1 = 'evaluateMVA("/home/fynu/swertz/scratch/CMSSW_7_6_3_patch2/src/cp3_llbb/HHTools/mvaTraining_hh/weights/BDTNAME_kBDT.weights.xml", '
+        baseStringForMVA_part1 = 'evaluateMVA("/home/fynu/sbrochet/scratch/Framework/CMSSW_7_6_5/src/cp3_llbb/HHTools//mvaTraining_hh/weights/BDTNAME_kBDT.weights.xml", '
```
Comment (swertz) [inline histFactory_hh/basePlotter.py:41]: I don't know how you feel about it but for the time being I'd prefer always keeping the previous path as a comment, it's easier to switch back if needed...

File: histFactory_hh/generatePlots.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def getBinningStrWithMax(nBins, start, end, max):
 includes.append( os.path.join(scriptDir, "..", "common", "reweight_v1tov3.h") )
 
 code_before_loop += """
-getBenchmarkReweighter("/home/fynu/sbrochet/scratch/Framework/CMSSW_8_0_6/src/cp3_llbb/HHTools/scripts/", 0, 11);
+getBenchmarkReweighter("/home/fynu/sbrochet/scratch/Framework/CMSSW_7_6_5/src/cp3_llbb/HHTools/scripts/", 0, 11);
```
Comment (swertz) [inline histFactory_hh/generatePlots.py:42]: same here

File: histFactory_hh/generatePlots.py [hunk 3]
```diff
@@ -78,26 +78,26 @@ def getBinningStrWithMax(nBins, start, end, max):
...
-        },
-        "10x25": {
-            "mjjBinning": getBinningStrWithMax(10, 0, 600, 13000), 
-            "bdtNbins": 25
-        },
-        "5x25": {
-            "mjjBinning": getBinningStrWithMax(5, 0, 600, 13000), 
-            "bdtNbins": 25
-        },
+        #"25x25": {
```
Comment (swertz) [inline histFactory_hh/generatePlots.py]: For those of us not using fancy vim plugins, it's really easier if the '#' are aligned ;)

File: histFactory_hh/basePlotter.py (hunk 1 above)
Comment (blinkseb) [inline histFactory_hh/basePlotter.py:41]: Well `git` is your friend here, no need to do the job it's already doing?

Comment (swertz) [inline histFactory_hh/basePlotter.py:41]: If you want to keep newer stuff, plots, ..., but just switch that it's easier to have it in the file. But it probably won't come up so fine with tme ;)

**Pull request merged**